plugin-modernizer-core/src/main/resources/update_center.properties

```diff
@@ -0,0 +1 @@
+update.center.url=https://updates.jenkins.io/current/update-center.actual.json
```

Comment:
Is better than hard-coding but I would prefer same approach that the plugin installation manager (at least checking the presence of the environment variable `JENKINS_UC`

https://github.com/jenkinsci/plugin-installation-manager-tool 

```
--jenkins-update-center: (optional) Sets the main update center filename, which can also be set via the JENKINS_UC environment variable. If a CLI option is entered, it will override what is set in the environment variable. If not set via CLI option or environment variable, will default to https://updates.jenkins.io/update-center.actual.json
```

Reply:
`--jenkins-update-center` is the option gets filename which is manually stored by user in local?

Reply:
https://github.com/jenkinsci/plugin-installation-manager-tool/blob/f36aacd44012c75dd735930270229ee2fce884da/plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/CliOptions.java#L102

It's an URL

Reply:
Added `JENKINS_UC` env in [cd7dfb3](https://github.com/jenkinsci/plugin-modernizer-tool/pull/87/commits/cd7dfb3f62f4907f63e8dc258885d009ab022463) will add new option (`--jenkins-update-center`) in next PR